### PR TITLE
MH-13459, Make Paella Use UI Configuration Service

### PR DESCRIPTION
**Pull request by lkiesow**
…this allows us to drop Paella's own configuration service.

### CODE REVIEW

File: docs/guides/admin/docs/releasenotes.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ A non-comprehensive list of improvements:
 Configuration changes
 ---------------------
 
-- …
+- The default configuration for the Paella player has been moved to `etc/ui-config/mh_default_org/paella/config.json`
```
Comment (miesgre) [inline docs/guides/admin/docs/releasenotes.md:41]: Should this be added also to `upgrade.md` file?
Why there is a `Configuration Changes` section in both files?

**Review by miesgre — CHANGES_REQUESTED** [review]
@lkiesow You can remove the `Paella player configuration folder` section from `etc/custom.properties` file also.

File: docs/guides/admin/docs/releasenotes.md (hunk 1 above)
Comment (lkiesow) [reply docs/guides/admin/docs/releasenotes.md:41]: The final docs are composed by the release managers anyway. Let them figure out the merge ;-)
It doesn't make sense to have it in both places.

**Comment by lkiesow** [comment]
Removed the option from the `custom.properties` and fixed the merge conflicts.

**Review by miesgre — APPROVED** [review]

**Comment by lkiesow** [comment]
Thanks for the review, @miesgre

**Pull request merged**